restassuredproject/src/main/java/endpoint/FindPetsByIdEndpoind.java

```diff
@@ -0,0 +1,26 @@
+package endpoint;
+
+import dto.ErrorMessage;
+import dto.RootObject;
+import io.restassured.response.Response;
+
+import static io.restassured.RestAssured.given;
+
+public class FindPetsByIdEndpoind {
+
+    public Response getFindById(String id) {
+        Response response = given().pathParam("pet", id).
+                when().get("http://petstore.swagger.io/v2/pet/{pet}");
```

Comment:
It's better to move domain to config part 
https://github.com/rest-assured/rest-assured/wiki/usage#configuration